docs/editor/settings-sync.md

```diff
@@ -195,6 +195,11 @@ It's possible that your wallet (aka keyring) is closed. If you open [KWalletMana
 
 If you are using KDE5 or higher and are having trouble connecting to `kwallet5` (like users of the unofficial VS Code Flatpak in [issue #189672](https://github.com/microsoft/vscode/issues/189672)), you can try [configuring the keyring](#other-linux-desktop-environments) to `gnome-libsecret` as this will use the [Secret Service API](https://www.gnu.org/software/emacs/manual/html_node/auth/Secret-Service-API.html) to communicate with any valid keyring. `kwallet5` implements the Secret Service API and can be accessed using this method.
 
+Another Flatpak/KDE5 fix you can try is granting the specific D-Bus service persmissions:
```

Comment:
```suggestion
If you're still experiencing trouble connecting to `kwallet5`, some users have reported that granting the specific D-Bus service permissions proved a viable fix:
```

Reply:
Sure, but maybe we can combine it with the suggested workaround in the paragraph above, which is also Flatpak and KDE related, and I think also not verified by the team?

Reply:
In a new section.

Reply:
Oh, that PR was a MS member.